python/cac_tripplanner/templates/home.html

```diff
@@ -62,8 +62,35 @@
     <div class="places">
         <div class="places-content">
             <header class="places-header">
-                <h1>Places we love</h1>
-                <a href="#" class="map-view-btn">Map View</a>
+                <div class="places-header-content">
+                    <h1>Places we love</h1>
+                    <a href="#" class="map-view-btn">Map View</a>
+                    <div class="filter-picker">
+                        <div class="filter-toggle">
+                            <div class="all filter-option on" title="All" data-filter="All">
+                                <span class="filter-label">All</span>
```

Comment:
Is there a structural need for `span.filter-label`?

Reply:
The class is there for styling, to match the usage from the mode selectors in the directions form above it. The class is not presently referenced in the JavaScript code, if that's what's meant by "structural."

Reply:
Great. That answers my question.